usecases/permissions_requests.go

```diff
@@ -0,0 +1,120 @@
+package usecases
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/topfreegames/Will.IAM/models"
+	"github.com/topfreegames/Will.IAM/repositories"
+)
+
+// PermissionsRequests define entrypoints for PermissionsRequests actions
+type PermissionsRequests interface {
+	Create(*models.PermissionRequest) error
+	Deny(string, string) error
```

Comment:
Ditto for (string, string) and others.

Reply:
I lost to what other comment the ditto refers

Reply:
It's for parameter names. (string, string) is too ambiguous.

Reply:
Done!